OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -178,5 +181,27 @@ void RevokeUpgrades(Actor self)
 				foreach (var u in Info.WhileCloakedUpgrades)
 					upgradeManager.RevokeUpgrade(self, u, this);
 		}
+
+		void INotifyHarvesterAction.MovingToResources(Actor self, CPos targetCell, Activity next){}
```

Comment:
Our coding style wants a space between the parens and the bracket, and also between the brackets, so it should look like `, Activity next) { }`

Same for the other occurrences below this line.